packages/ui/src/views/repo/repo-summary/components/clone-repo-dialog.tsx

```diff
@@ -0,0 +1,122 @@
+import { useState } from 'react'
+
+import {
+  Button,
+  CopyButton,
+  DropdownMenu,
+  DropdownMenuContent,
+  DropdownMenuItem,
+  DropdownMenuTrigger,
+  Icon,
+  Input,
+  Tabs,
+  TabsList,
+  TabsTrigger
+} from '@/components'
+import { TranslationStore } from '@/views'
+
+export interface CloneRepoDialogProps {
+  sshUrl: string
+  httpsUrl: string
+  handleCreateToken: () => void
+  useTranslationStore: () => TranslationStore
+}
+
+export enum CloneRepoTabs {
+  HTTPS = 'https',
+  SSH = 'ssh'
+}
+
+export const CloneRepoDialog: React.FC<CloneRepoDialogProps> = ({
+  httpsUrl,
+  sshUrl,
+  handleCreateToken,
+  useTranslationStore
+}) => {
+  const [currentTab, setCurrentTab] = useState(CloneRepoTabs.HTTPS)
+  const { t } = useTranslationStore()
+
+  return (
+    <DropdownMenu>
+      <DropdownMenuTrigger asChild>
+        <Button>
+          <Icon name="clone" />
+          &nbsp; {t('views:repos.clone', 'Clone')}
+        </Button>
+      </DropdownMenuTrigger>
+      <DropdownMenuContent className="w-[328px] p-0" align="end">
+        <div className="px-3 pt-2">
```

Comment:
```suggestion
        <div className="px-4 pt-4">
```
It seems that the Tabs component also needs updated padding—currently, it’s set to `px-3` by default, but it should be `px-4`

<img width="564" alt="image" src="https://github.com/user-attachments/assets/966f24db-535e-45b9-ab2a-a78aa34bf21c" />